server/plugin/virtual_agent.go

```diff
@@ -321,11 +321,11 @@ func (p *Plugin) createMessageAttachment(fileID string) (*MessageAttachment, err
 	var attachment *MessageAttachment
 	fileInfo, appErr := p.API.GetFileInfo(fileID)
 	if appErr != nil {
-		return nil, fmt.Errorf("error getting file info. Error: %s", appErr.Error())
+		return nil, fmt.Errorf("error getting file info. Error: %w", appErr)
 	}
 
 	//TODO: Add a configuration setting for expiry time
-	expiryTime := time.Now().UTC().Add(time.Second * 30)
+	expiryTime := time.Now().UTC().Add(time.Minute * 15)
```

Comment:
Create a constant variable for 15 called `AttachmentLinkExpiryTimeInMinutes` and use that here.